cmd/kubermatic-api/main.go

```diff
@@ -14,24 +14,29 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package classification Kubermatic API.
+// Kubermatic Kubernetes Platform API
 //
-// Kubermatic API
+// This spec describes possible operations which can be made against the Kubermatic Kubernetes Platform API.
 //
-// This describes possible operations which can be made against the Kubermatic API.
-//
-// Terms Of Service:
-//
-// There are no TOS at this moment, use at your own risk we take no responsibility
-//
-//     Version: 2.11
+//     Schemes: https
+//     Host: dev.kubermatic.io
```

Comment:
This one is the difference from before. Previously this was only in `routes_v1.go` and used only during API client generation. Do we really want to hide that in our main `swagger.json`?